Guided-registration/README.md

```diff
@@ -0,0 +1,94 @@
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://thin-edge.io">
+    <img src="img/logo.png" alt="Logo">
+  </a>
+
+  <h3 align="center">dockerized thin-edge.io example application </h3>
+  <p align="center"><a href="https://thin-edge.io">thin-edge.io</a></p>
+  <br />
+  <br />
+</p>
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#about">About</a></li>
+    <li><a href="#getting started">Getting Started</a></li>
```

Comment:
```suggestion
    <li><a href="#getting-started">Getting Started</a></li>
```
Link is currently broken, I think the anchor cannot contain space character

Reply:
corrected